src/v2/view-builder/views/ov/ChallengeAuthenticatorDataOktaVerifyView.js

```diff
@@ -7,7 +7,42 @@ import { Body as SelectAuthenticatorVerifyViewBody } from '../SelectAuthenticato
 import { AUTHENTICATOR_KEY } from '../../../ion/RemediationConstants';
 
 const Body = SelectAuthenticatorVerifyViewBody.extend({
+  initialize() {
+    BaseForm.prototype.initialize.apply(this, arguments);
+    if (this.isPushOnlyFlow()) {
+      this.addView();
+    }
+  },
+
+  addView() {
+    this.add(createButton({
+      className: 'button button-wide button-primary send-push',
+      title: loc('oie.okta_verify.sendPushButton', 'login'),
+      click: () => {
+        this.$el.submit();
+      },
+    }));
+    this.add(
+      `<span class='accessibility-text' role='alert'>${loc('oie.okta_verify.sendPushButton', 'login')}</span>`,
+    );
+  },
+
+  postRender() {
+    if (this.isPushOnlyFlow()) {
+      this.$el.addClass('okta-verify-push-challenge');
+      this.$el.find('.okta-form-subtitle').hide();
+      // Move checkbox below the button
+      const checkbox = this.$el.find('.o-form-fieldset');
+      checkbox.length && this.$el.find('.o-form-fieldset-container').append(checkbox);
+    }
+  },
+
   getUISchema() {
+    if (this.isPushOnlyFlow()) {
+      // Prevent from displaying radio buttons on the UI
+      const uiSchemas = BaseForm.prototype.getUISchema.apply(this, arguments);
```

Comment:
nit: since it is the same logic and lines below, maybe we can do 
```
// Change the UI schema to not display radios here.
const uiSchemas = BaseForm.prototype.getUISchema.apply(this, arguments);
const methodsSchema = uiSchemas.find(schema => schema.name === 'authenticator.methodType');
if (this.isPushOnlyFlow()) {
  // Only one item, so no need to sort
  return methodsSchema;
}
```
Although this might change if we extend `BaseForm` per my previous comment

Reply:
this piece of code will also be moved to the new view. Also, its needed to filter out `authenticator.methodType` otherwise radio button would be displayed on the UI for 'Get a push notification' type.